.pre-commit-config.yaml

```
@@ -8,7 +8,7 @@ repos:
         stages: [commit]
         args: ["--config", "pyproject.toml", "tests", "src", "benchmarks", "metrics"]
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: 'v0.0.255'
+    rev: 'v0.0.278'
```

Hi @polinaeterna @lhoestq as this has not been merged yet and `ruff v0.0.286` has been recently released (see https://github.com/astral-sh/ruff/releases/tag/v0.0.286), shouldn't we upgrade it before merging (as this PR looks ready), thanks!

```suggestion
    rev: 'v0.0.286'
```